packages/hash/api/src/db/postgres/history.ts

```diff
@@ -167,7 +167,9 @@ export const getImpliedEntityHistory = async (
             accountId: link.sourceAccountId,
             entityId: link.sourceEntityId,
             entityVersionId: link.destinationEntityVersionId,
-            validForSourceEntityVersionIds: link.sourceEntityVersionIds,
+            /** @todo: fix this type when fixing implied history resolver */
+            validForSourceEntityVersionIds: (link as any)
+              .sourceEntityVersionIds,
```

Comment:
TS is complaining here because this PR removes `sourceEntityVersionIds` - opted to cast to fix the TS error, but this needs to change as part of the upcoming task where we fix the history resolvers the whole history